tools/applicationinsights-web-snippet/src/applicationinsights-web-snippet.ts

```diff
@@ -10,4 +10,20 @@ function webSnippetVersion() {
     return "";
 }
 
-export { webSnippet, webSnippetCs, webSnippetVersion }
\ No newline at end of file
+function substituteInstrumentationKey(key: string) {
```

Comment:
Lets call these something like

`getSdkLoaderScript(config: SdkLoaderConfig )`
And define a specific config for this that

```ts
export interface SdkLoaderConfig {
<include the instrumentation key and the connection string>
}
```

Where this configuration could (or will eventually) support all of the snippet configurations -- so we could even support the default snippet configuration.